Changelog.md

```diff
@@ -11,6 +11,7 @@ Bugfixes:
  * Commandline interface: Do not try creating paths ``.`` and ``..``.
  * Type system: Fix a crash caused by continuing on fatal errors in the code.
  * Type system: Disallow arrays with negative length.
+ * Inline assembly: Charge one stack slot for non-value types during analysis.
```

Comment:
Actually the fix is to charge two stack slots for value-types during analysis (at least the test reads like that).